From eaf7106bf4e4100df0c91daea2a8a20cddc22886 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 7 Mar 2024 23:30:04 +0800
Subject: [PATCH] chore(src/org.kde.kamoso.desktop): move category to Utility

This is not so much a multimedia application but a tool to access the webcam.
---
 src/org.kde.kamoso.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/org.kde.kamoso.desktop b/src/org.kde.kamoso.desktop
index 254156c..e6d5ee8 100755
--- a/src/org.kde.kamoso.desktop
+++ b/src/org.kde.kamoso.desktop
@@ -141,4 +141,4 @@ Exec=kamoso -qwindowtitle %c
 Icon=kamoso
 Type=Application
 X-DocPath=kamoso/index.html
-Categories=Qt;KDE;AudioVideo;
+Categories=Qt;KDE;Utility;
-- 
2.44.0

